pkg/fakerp/dns.go

```diff
@@ -123,15 +125,19 @@ func DeleteOCPDNS(ctx context.Context, subscriptionID, resourceGroup, dnsResourc
 	zc.Authorizer = authorizer
 	rsc := dns.NewRecordSetsClient(subscriptionID)
 	rsc.Authorizer = authorizer
+	count := new(int32)
+	*count = 100
+	zoneResults, err := zc.ListByResourceGroup(ctx, resourceGroup, count)
```

Comment:
Before this PR, when the zoneName were well known, removing zones was easier.  Now with a randomized string we need to list the dns zones from the resource group and then remove them from the main zone record.